roles/network_plugin/calico/templates/calico-typha.yml.j2

```diff
@@ -136,11 +136,10 @@ spec:
             name: cacert
             readOnly: true
 {% endif %}
-          # Needed for version >=3.7 when the 'host-local' ipam is used
-          # Should never happen given templates/cni-calico.conflist.j2
-          # Configure route aggregation based on pod CIDR.
-          # - name: USE_POD_CIDR
-          #   value: "true"
+{% if calico_ipam_host_local %}
```

Comment:
It looks like there's some formatting error. doesn't it?

Reply:
I've reviewed it again, and everything seems correct on my end. If you notice any issues or have any specific suggestions, please feel free to point them out.

Reply:
HI @cyclinder 
Do you have any comments ?

Reply:
@yankay sorry for the delay.

> and everything seems correct on my end

aha, it's maybe a github issue.